let-talk-rxjs.md

```diff
@@ -4,30 +4,30 @@ Title: Lets Talk RxJS Operators
 
 ## Let's Talk About RxJS Operators
 
-To be able to first understand Operators, you first need to understand what Observables are. "An Observable is basically a function that can return a stream of values to an observer over time, this can either be synchronously or asynchronously." In vanilla JavaScript, they can easily be compared to Arrays. 
```

Comment:
There were repeated words in the first sentence which sounds a little awkward when reading.

I cut the second `first` as it's implied and switched `understand` -> `grok` (feel free to change that to something else)

Reply:
is the sentence in quotes a quote from somewhere? if not, should remove quotes

Would love to see you rephrase this sentence in your own words afterwards

Reply:
words like 'easily' should generally be avoided. This might be easy for you but others might not see the similarities